fix(curriculum): Clarify quiz game instructions and tests (#59355)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/lab-quiz-game/66f17db06803d11a1bd19a20.md

1. You should have a function named `getRandomQuestion` that takes an array of questions as a parameter and returns a random question object from the array.
1. You should have a function named `getRandomComputerChoice` that takes the array of the available choices as a parameter, and returns a random answer to the selected question.

> 1. You should have a function named `getResults` that takes the selected question object and the computer choice as its parameters and returns `The computer's choice is correct!` if the answer is correct. Otherwise, it returns `The computer's choice is wrong. The correct answer is: <correct-answer>`, where `<correct-answer>` is the value of the correct answer to the chosen question.
> 1. Your `getResults` function should take the question object and computer choice as its parameters, in order. 

Reviewer: This is a little redundant with the two user stories. This can be combined.

```suggestion
1. You should have a function named `getResults` that takes the question object as the first parameter and the computer's choice as the second parameter. The function should return `The computer's choice is correct!` if the answer is correct. Otherwise, it returns `The computer's choice is wrong. The correct answer is: <correct-answer>`, where `<correct-answer>` is the value of the correct answer to the chosen question.
```

Author: Yeah I kinda thought the same but figured the instructions would be too long. I'll change it though

assert.isFunction(getResults);
```

> Your `getResults` function should take the question object and computer choice as parameters, in that order.

Reviewer: ```suggestion
Your `getResults` function should take the question object as the first parameter and the computer's choice as the second parameter.
```
